src/plugins/data_source_management/public/service/creation/config.ts

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
don't think we need this file, please try remove it

Reply:
done